fetcher.sh

```diff
@@ -104,32 +102,26 @@ if [ "$kernel" = "Linux" ]; then
 		esac
 	}
 
-	# editor, remove the file path
-	EDITOR="${EDITOR##*/}"
-
-	# terminal, remove declaration of color support from the name
+	# Terminal, list running processes and check for common terms
 	term=$(ps -e | grep -m 1 -o \
 		-e " alacritty$" \
 		-e " kitty$" \
 		-e " xterm$" \
-		-e " urxvt$" \
-		-e " xfce4-terminal$" \
-		-e " gnome-terminal$" \
-		-e " mate-terminal$" \
+		-e " u*rxvt[dc]*$" \
+		-e " [a-z0-9-]*terminal$" \
```

Comment:
liberal patterns like these should probably be tried when every conservative pattern fails, i'd move this to line 120

Reply:
I don't think where it is would affect it.

Reply:
Grep would be horribly inefficient if it repeated itself for every arg.